src/app/components/team/Statuses/EditStatusDialog.js

```diff
@@ -26,11 +24,18 @@ const EditStatusDialog = ({
 }) => {
   const [statusLabel, setStatusLabel] = React.useState(status ? status.label : '');
   const [statusDescription, setStatusDescription] = React.useState(status ? status.locales[defaultLanguage].description : '');
-  const [statusColor, setStatusColor] = React.useState(status ? status.style.color : 'var(--textPrimary)');
+  const [statusColor, setStatusColor] = React.useState(status ? status.style.color : '#567bff');
```

Comment:
for reviewers, I noticed that we were using a css var before, but the color picker doesn't correctly set the color unless you use a hex code, just an FYI for this change